lib/nerves_hub/http_fwup_stream.ex

```diff
@@ -144,6 +144,12 @@ defmodule NervesHub.HTTPFwupStream do
     {:stop, {:error, {:http_error, {status_code, body}}}, s}
   end
 
+  @impl true
+  def handle_info({:http, {_ref, {:error, error}}}, state) do
+    Logger.error("FWUP Error: #{inspect(error)}")
```

Comment:
Can we indicate that this is an HTTP not just a generic FWUP error?

Reply:
So its next stop after this logger is _another_ logger in the channel:
https://github.com/nerves-hub/nerves_hub/blob/master/lib/nerves_hub/channel.ex#L76

I went with `FWUP` just because that is the module it is in and the next log output will say HTTP. I also think all these log messages need a good overhaul (we are duplicating log lines everywhere, sending it to other modules to print again with a weird message, etc etc..makes debugging a little weird). Was just making an issue actually...

Maybe I change this location to be the HTTP log message, and remove the one in the `Channel` module?

Reply:
Totally agree that log messages need an overhaul.

Yes, for this PR, I like the idea of changing this error to an HTTP one and removing that one in `Channel`.

Reply:
Changed